.cspell.json

```diff
@@ -33,6 +33,7 @@
     "UBIQUIBOT",
     "URLSAFE",
     "WXDAI",
+    "wfzpewmlyiozupulbuur",
```

Comment:
This should probably be added as a directive in the file instead: 

cspell: ignore wfzp...

Reply:
Updated https://github.com/ubiquity/onboard.ubq.fi/pull/17/commits/4a54593f65a3804ec3e4d83222f167a17dbad12e